### ISSUE TITLE
Science Lab: Antimatter mg/day

### ISSUE THREAD

**Issue by caipi-** [FractalUK/KSPInterstellar#36]
The science lab says that I produce 27,614mg Antimatter per day.
However I only produce around 6,9mg in one Kerbal day.

The display mg/day is for an Earth day (24h). A day on Kerbin is merely 6h long.This should in my opinion be adjusted to a Kerbin day.

I assume the same goes for the Antimatter Collector and the Science/day

**Comment by confusingbits** [comment]
Yes, all day calculations are done in seconds, which do not change between earth or kerbin time. There are a lot of calculations done based 86400 seconds. So much so that I wouldn't expect it to change any time soon.

**Comment by ruifung** [comment]
Well, I've gotten used to multiplying any per day rate from KSPI by 0.25 so I get the rate per kerbin day.
